pipelines/components/image_classifier_keras/README.md

```diff
@@ -0,0 +1,131 @@
+
+
+# ML Pipeline - Image Classification using Keras
+
+## License
+
+Copyright 2018 Google LLC. All Rights Reserved.<br/>
```

Comment:
I don't think we need this header in the README file.

Reply:
+1 no need for License in README

Reply:
License removed from README